crates/live_update/src/jni.rs

```diff
@@ -22,20 +22,44 @@ use jni::{JNIEnv, JavaVM};
 use log::error;
 use log::LevelFilter;
 
-use figma_import::{fetch_doc, ConvertRequest};
+use figma_import::{fetch_doc, ConvertRequest, ProxyConfig};
 
 use crate::error_map::map_err_to_exception;
 
 fn throw_basic_exception(env: &mut JNIEnv, msg: String) {
-    //An error occurring while trying to throw an exception shouldn't happen, but let's at least panic with a decent error message
+    // An error occurring while trying to throw an exception shouldn't happen,
+    // but let's at least panic with a decent error message
     env.throw(msg).expect("Error while trying to throw Exception");
 }
 
+fn get_string(env: &mut JNIEnv, obj: &JObject) -> Option<String> {
+    match env.get_string(obj.into()) {
```

Comment:
I'm not sure how jni-rs works (we updated from the old one that I knew a bit better ;)), but I'm assuming all of these gets are local refs, so we won't get errors from exhausting global refs etc...